src/mixins/view.js

```diff
@@ -163,11 +167,18 @@ export default {
     // https://github.com/jashkenas/backbone/blob/1.2.3/backbone.js#L1235
     this._removeElement();
 
+    if (shouldTriggerDetach) {
+      this._isAttached = false;
+      this.triggerMethod('detach', this);
+    }
+
     // remove children after the remove to prevent extra paints
     this._removeChildren();
 
     this._destroyBehaviors(args);
 
+    this._isDestroyed = true;
```

Comment:
possible we want to make sure the view is destroyed prior to the behaviors?  Not 100% sure, but the behaviors know about the view... don't know if there's ever a use case for the behavior understanding if it is being destroyed on its own or as part of the view being destroyed.  Actually it might be reasonable to put this before `_removeChildren` as well.  if a childView triggers an event during a destroy that the parent is listening to, it may want to know if it is being destroyed or not.